openhabian-setup.sh

```diff
@@ -17,9 +17,12 @@ while [ -h "$SOURCE" ]; do
   [[ $SOURCE != /* ]] && SOURCE="$BASEDIR/$SOURCE"
 done
 BASEDIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
-SCRIPTNAME="$(basename $SOURCE)"
+# shellcheck disable=SC2034
```

Comment:
Please do one global exception for the whole file in the top.

Reply:
done